src/jit/rangecheck.cpp

```diff
@@ -875,29 +859,15 @@ Range RangeCheck::ComputeRangeForLocalDef(BasicBlock*          block,
         JITDUMP("----------------------------------------------------\n");
     }
 #endif
-    switch (asg->OperGet())
+    Range range = GetRange(asgBlock, asg->gtGetOp2(), monotonic DEBUGARG(indent));
```

Comment:
Could you please add an assert that `asg->OperIs(GT_ASG)` just to make my paranoia calm?

Reply:
Seems a bit redundant but I'll add it. Hopefully `GT_ASG` will disappear one day and take the assert with it :)